2.6/root/usr/share/container-scripts/mongodb/replset-functions.sh

```diff
@@ -0,0 +1,97 @@
+# This file contains functions	 for replSet manipulation
+
+# cache_container_addr waits till the container gets the external IP address and
+# cache it to disk
+function cache_container_addr() {
+  echo -n "=> Waiting for container IP address ..."
+  local i
+  for i in $(seq ${MAX_ATTEMPTS}); do
+    if ip -oneline -4 addr show up scope global | grep -Eo '[0-9]{,3}(\.[0-9]{,3}){3}' > "${HOME}"/.address; then
+      echo " $(mongo_addr)"
+      return 0
+    fi
+    sleep ${SLEEP_TIME}
+  done
+  echo "Failed to get Docker container IP address." && exit 1
```

Comment:
I don't know. Also copied from master branch.

I make sense to me to have `exit` on separate line. Changed.